frontend/packages/pipelines-plugin/src/utils/pipeline-utils.ts

```diff
@@ -528,3 +528,21 @@ export const returnValidTaskModel = (task: TaskKind): K8sModel => {
   }
   return TaskModel;
 };
+
+export enum TaskRunResultsAnnotations {
+  KEY = 'task.results.key',
+}
+
+export enum TaskRunResults {
+  SBOM = 'LINK_TO_SBOM',
+  SCAN_OUTPUT = 'SCAN_OUTPUT',
+  TEST_OUTPUT = 'TEST_OUTPUT',
+}
+
+export const getSbomTaskRun = (taskruns: TaskRunKind[]): TaskRunKind =>
+  taskruns?.find(
```

Comment:
We don't read `task.results.format` anywhere do we ?

Reply:
You are right @vdemeester , This PR just surfaces the SBOM link and reads annotations. The output tab PR will be using all other annotations like format, type etc

Reply:
The CVE summary is also JSON based, is that part of this PR?